ci-operator/templates/cluster-launch-installer-e2e.yaml

```diff
@@ -247,6 +293,17 @@ objects:
         fi
         cp $KUBECONFIG /tmp/admin.kubeconfig
         echo "Installation successful"
+        echo "Starting installer smoke tests..."
+        export SMOKE_KUBECONFIG=${KUBECONFIG}
+        export SMOKE_MANIFEST_PATHS=/tmp/cluster/generated/manifests
+        # 3 masters/3 workers/1 bootstrap(if post remove etcd)
```

Comment:
We can remove "(if post remove etcd)", since both this repo and the installer repo are past that point.

Reply:
ok, there was talk that the bootstrap instance would eventually not be included in the overall node count though? if it's terminated after the bootstrap is complete? we'll have to keep an eye on that number, it definitely shouldn't be hardcoded the way it is